cluster/helm/splice-sv-node/templates/sv.yaml

```diff
@@ -108,6 +108,14 @@ spec:
               value: {{ .Values.auth.audience | quote }}
             - name: SPLICE_APP_SV_AUTH_JWKS_URL
               value: {{ .Values.auth.jwksUrl | quote }}
+            {{ if .Values.auth.connectionTimeout }}
```

Comment:
you made the mistake of adding this so now I'm obviously gonna ask for a helm unittest :) same for the others

Reply:
https://github.com/hyperledger-labs/splice/blob/main/TESTING.md#helm-checks